### ISSUE TITLE
ADD: Remember Password for 15 Mins

### ISSUE THREAD

**Issue by thorchain-admin** [thorchain/asgardex-electron#1308]
# Checkbox option, means no password entry for 15 mins. 

Put it under the field

[ ] Remember Password for 15 Mins

<img width="631" alt="image" src="https://user-images.githubusercontent.com/41812729/114391986-849da880-9bdb-11eb-95b6-21a29307789a.png">

**Comment by thirdEyeGorilla** [comment]
Just looking for clarification on this issue.  Would you still like the Modal to display with the password prefilled or skip the modal? I'm assuming you would like it not to be displayed after the checkbox is ticked?

Screenshot attached of WIP.
![Screen Shot 2021-08-15 at 11 59 02 am](https://user-images.githubusercontent.com/37689057/129465062-d414182b-1ceb-48a0-975e-32e9fc0696db.png)

**Comment by veado** [comment]
> @veado Just looking for clarification on this issue. Do you still want the modal to appear with the password prefilled?
Yep, changes for the modal are correct (just a typo: "save" -> "remember" - see issue description). Note: The password will never be saved in memory or on disc, but a counter will run for 15 min "behind the scenes" to hide this modal. That's any transaction can be sent without showing the modal to enter a password.

**Comment by daryazata** [comment]
@veado this is not yet resolved right? if no, do I need some sort of test login data to work on this issue?

**Comment by veado** [comment]
@daryazata Feel free to grab it, it's still open. No test login data needed, you just need to create your own (test) account by using ASGARDEX for the first time. For further questions, pls join [Discord (#asgardex-desktop channel)](https://discord.gg/pHcS67yX7Z).
